src/components/common/UserDetails.tsx

```
@@ -49,6 +50,23 @@ const Field: React.FC<FieldProps> = ({ label, value, defaultValue = '-' }) => (
 	</Box>
 );
 
+// Helper function to process field values based on field type
+const processFieldValue = (field: CustomField, userDob?: string | null): string | number | null => {
```

ESLint

[error] Replace `field:·CustomField,·userDob?:·string·|·null` with `⏎↹field:·CustomField,⏎↹userDob?:·string·|·null⏎`

(prettier/prettier)